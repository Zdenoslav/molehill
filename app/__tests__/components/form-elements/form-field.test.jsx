import React from 'react';
import { shallow } from 'enzyme';

import FormField from 'app/components/form-elements/form-field';

describe('FormField', () => {

  it('should render FormField', () => {
    const wrapper = shallow(<FormField />);
    expect(wrapper).toMatchSnapshot();
  });

});
