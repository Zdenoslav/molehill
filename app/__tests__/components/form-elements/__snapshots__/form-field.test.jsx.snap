// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormField should render FormField 1`] = `
<styled.div>
  <Field />
</styled.div>
`;
